scalability_and_performance/psap-node-feature-discovery-operator.adoc

```diff
@@ -0,0 +1,16 @@
+[id="node-feature-discovery-operator"]
+= The Node Feature Discovery Operator
+include::modules/common-attributes.adoc[]
+:context: node-feature-discovery-operator
+
+toc::[]
+
+Learn about the Node Feature Discovery Operator and how you can use it to expose node-level information by orchestrating the Node Feature Discovery, a Kubernetes add-on for detecting hardware features and system configuration.
```

Comment:
Node Feature Discovery - I think the acronym NFD can be introduced here as "Node Feature Discovery (NFD)" and then the rest of docs should use NFD and not "Node Feature Discovery (NFD)". @ArangoGutierrez